docs/loading/Flink-connector-starrocks.md

```diff
@@ -1,173 +1,647 @@
+
+    ```SQL
+    CREATE TABLE `score_board` (
+        `id` INT,
+        `name` STRING,
+        PRIMARY KEY (id) NOT ENFORCED
+    ) WITH (
+        'connector' = 'starrocks',
+        'jdbc-url' = 'jdbc:mysql://127.0.0.1:9030',
+        'load-url' = '127.0.0.1:8030',
+        'database-name' = 'test',
+        'table-name' = 'score_board',
+        'username' = 'root',
+        'password' = '',
+        'sink.properties.partial_update' = 'true',
+        -- only need for connector version <= 1.2.7
+        'sink.properties.columns' = 'id,name,__op'
+    ); 
+    ```
+
+3. Insert data to the table in Flink SQL client, and only update the column `name`
+
+    ```SQL
+    INSERT INTO `score_board` VALUES (1, 'starrocks-update'), (2, 'flink-update');
+    ```
+
+4. Query the StarRocks table in mysql client
+  You can see that only values for `name` changes, and the values for `score` does not change.
+  
+    ```SQL
+    mysql> select * from score_board;
+    +------+------------------+-------+
+    | id   | name             | score |
+    +------+------------------+-------+
+    |    1 | starrocks-update |   100 |
+    |    2 | flink-update     |   100 |
+    +------+------------------+-------+
+    2 rows in set (0.02 sec)
+    ```
+
+#### Conditional update
+
+This example will show how to do conditional update according to the value of column `score`. The update for an `id`
+takes effect only when the new value for `score` is has a greater or equal to the old value.
+
+1. Insert initial data to StarRocks table in MySQL client
+
+    ```SQL
+    mysql> INSERT INTO `score_board` VALUES (1, 'starrocks', 100), (2, 'flink', 100);
+    
+    mysql> select * from score_board;
+    +------+-----------+-------+
+    | id   | name      | score |
+    +------+-----------+-------+
+    |    1 | starrocks |   100 |
+    |    2 | flink     |   100 |
+    +------+-----------+-------+
+    2 rows in set (0.02 sec)
+    ```
+
+2. Create a Flink table `score_board` in the following ways
+  
+    - Define the DDL including all of columns
+    - Set the option `sink.properties.merge_condition` to `score` which tells the connector to use the column `score`
+    as the condition
+
+    ```SQL
+    CREATE TABLE `score_board` (
+        `id` INT,
+        `name` STRING,
+        `score` INT,
+        PRIMARY KEY (id) NOT ENFORCED
+    ) WITH (
+        'connector' = 'starrocks',
+        'jdbc-url' = 'jdbc:mysql://127.0.0.1:9030',
+        'load-url' = '127.0.0.1:8030',
+        'database-name' = 'test',
+        'table-name' = 'score_board',
+        'username' = 'root',
+        'password' = '',
+        'sink.properties.merge_condition' = 'score'
+    );
+    ```
+
+3. Insert data to the table in Flink SQL client, and update id 1 with a smaller score, and id 2 with a larger score.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/4b0781be-12b2-4134-96f1-c66679933b09)
the sentence does not make sense.